### DISCUSSION TITLE
듀얼부팅관련

### DISCUSSION THREAD

**Discussion by uijinee** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#51] [2024년도 기타 질문]
1. 꼭 우분투22.04를 듀얼부팅으로 설치해야하나요? vmware같은 가상환경에 설치해도 되는지 궁금합니다.
2. 우분투를 노트북에 설치하고 22일부터 진행되는 자율주행 통합과정에서는 이 환경을 그대로 사용하는건가요?
    노트북 용량이 부족해서 데스크톱에 설치하고자 통합과정을 진행하는데 문제가 없을지 궁금합니다.

**Comment by Jinsun-Lee** [comment]
1. 듀얼 부팅 설치 방법을 안내한 이유는 발생 가능한 문제의 경우의 수를 제한하고, 가상화 시스템으로 인한 문제를 배제하기 위해서입니다.
 과제 수행에 문제가 없다면 VMware를 사용하셔도 됩니다. 그러나 VMware를 사용하여 생긴 오류는 도움을 드리기 어렵습니다. 
 
---
 
2. 모든 학생들에게 "동일한" 고사양 노트북을 제공합니다. 개발환경은 Ubuntu 22.04와 ROS2 Humble을 사용합니다.

---

3. 위 답변 참고 바랍니다.
